modules/gathering-data-specific-features.adoc

```diff
@@ -32,7 +32,7 @@ ifndef::openshift-origin[]
 |`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`
 |Data collection for OpenShift Serverless.
 
-|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7`
+|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8`
 |Data collection for Red Hat OpenShift Service Mesh.
 
 |`registry.redhat.io/rhcam-1-2/openshift-migration-must-gather-rhel8`
```

Comment:
Sorry I can't comment on the actual line in `gathering data about specific features` section. The first `note` right before the prereqs, you say, `...add the `--image gather` argument.` Maybe say, `...add the argument, `--image gather.` 

This is totally a preference things. Whenever there's something the user might copy and paste or click, I tend to link to put it at the end of the sentence to make it easier to copy or click. Again, totally up to you and it's good as is. Simply a suggestion, thanks!